lib/main.js

```diff
@@ -133,6 +135,22 @@ if (registerCommands) {
     });
 }
 
+let steps;
+function testSteps(){
+  const StepsClass =  new TestSteps();
+
+  const stepsObj = {};
+  stepsObj.describe = StepsClass.describe.bind(StepsClass);
+  stepsObj.it = StepsClass.it.bind(StepsClass);
+  stepsObj.before = StepsClass.before.bind(StepsClass);
+  stepsObj.after = StepsClass.after.bind(StepsClass);
+  stepsObj.beforeEach = StepsClass.beforeEach.bind(StepsClass);
+  stepsObj.afterEach = StepsClass.afterEach.bind(StepsClass);
+
+  steps = stepsObj;
+}
+testSteps();
+
```

Comment:
`StepsClass` is not a class it's an object. And in JS convention it should be camel cased. This also seems unnecessarily complicated.

```
const testSteps = new TestSteps();

module.exports.describe = testSteps.describe.bind(testSteps);
...
```